phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/default/main.less

```diff
@@ -602,6 +602,10 @@
   -fx-font-family: "Roboto Light";
 }
 
+.input:error {
+  -fx-background-color: #f31111;
```

Comment:
Just asking:
Maybe a red border looks better?

Reply:
I tried but it doesn't change the border color of the existing border. Instead it adds a new rectangular border around it. Looks really strange.

Reply:
Ah wait. I used the wrong field in the CSS.